js/sign/src/cli.ts

```diff
@@ -0,0 +1,49 @@
+import commander from 'commander';
+import { IntegrityBlockSigner } from './integrityblock.js';
+import * as fs from 'fs';
+import crypto, { KeyObject } from 'crypto';
+
+function readOptions() {
+  return commander
+    .requiredOption(
+      '-i, --input <file>',
+      'input unsigned web bundle (required)'
```

Comment:
Does input have to be unsigned? It may be useful if one could specify a signed bundle and `wbn-sign` would push a signature to its signature stack.

Reply:
I think in the first place we only sign unsigned bundles but it doesnt harm to have it more general on this so I updated the text to be  'input web bundle to be signed (required)' :)